sphinx/templates/latex/latex.tex_t

```diff
@@ -30,6 +30,8 @@
 <%= contentsname %>
 <%= numfig_format %>
 <%= pageautorefname %>
+\newcommand{\sphinxVerbatimContinuedText}{\footnotesize(<%= _('continued from previous page') %>)}
+\newcommand{\sphinxVerbatimContinuesText}{\footnotesize(<%= _('continues on next page') %>)}
```

Comment:
as indicated in #3791 it seems gettext did not pick strings from longtable templates so I guess one will have to be careful about latex template as well